ts/src/base/Exchange.ts

```diff
@@ -2874,15 +2874,45 @@ export default class Exchange {
 
     getDefaultOptions () {
         return {
-            'defaultNetworkCodeReplacements': {
-                'ETH': { 'ERC20': 'ETH' },
-                'TRX': { 'TRC20': 'TRX' },
-                'CRO': { 'CRC20': 'CRONOS' },
-                'BRC20': { 'BRC20': 'BTC' },
-            },
```

Comment:
we no longer have "defaultNetworkCodeRepalcements" option, instead we have another (a bit more readable) dict